superset-frontend/src/components/ListView/Filters.tsx

```diff
@@ -145,8 +145,11 @@ function SelectFilter({
           data-test="filters-select"
           themeConfig={filterSelectTheme}
           stylesConfig={filterSelectStyles}
+          // @ts-ignore
```

Comment:
It seems I introduced these issues when I added https://github.com/vtaits/react-select-async-paginate in https://github.com/apache/incubator-superset/pull/10035 and this PR merely uncovered them (for some reason... 🤔). I'll address in another PR if it's possible, the types for that package are quite confusing.